observability/metrics/src/main/scala/com/daml/metrics/HealthMetrics.scala

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import com.daml.metrics.api.MetricHandle.Factory
+import com.daml.metrics.api.{MetricsContext, MetricName => MN}
```

Comment:
Why do we need this alias?

Reply:
Because I use `MetricName` as a constant for the name of the metric.